cluster-scope/overlays/moc/kustomization.yaml

```diff
@@ -37,6 +37,16 @@ resources:
   - ../../base/namespaces/thoth-graph-prod
   - ../../base/namespaces/thoth-infra-prod
   - ../../base/namespaces/thoth-middletier-prod
+  # Elastic Search Operator & Cluster Logging Operator
+  # Details: ( https://docs.openshift.com/container-platform/4.6/logging/cluster-logging-deploying.html#cluster-logging-deploy-cli_cluster-logging-deploying )
+  - ../../base/namespaces/openshift-operators-redhat
+  - ../../base/namespaces/openshift-logging
+  - ../../base/operator-groups/elastic-operator
```

Comment:
Few suggestions:
1. Can you please rename the `operator-groups` to be `operatorgroups`? This way it would match the "plural name" for the resource kind.
2. Can you please use the alphabetical ordering of the resources? We don't keep them together by their usage, but rather make them easier to locate in the repo.

Reply:
(1) Done
(2) Done